packages/bridge-ui/src/components/form/BridgeForm.svelte

```diff
@@ -273,7 +275,7 @@
       <button class="label-text" on:click={useFullAmount}
         >{$_("bridgeForm.maxLabel")}
         {tokenBalance.length > 10
-          ? `${truncateString(tokenBalance)}...`
+          ? `${truncateString(tokenBalance, 6)}...`
```

Comment:
A better approach is to use `toFixed(4)`

https://ethereum.stackexchange.com/questions/84004/ethers-formatetherwei-with-max-4-decimal-places

Reply:
I see some trade-offs here.
1. `tokenBalance` contains `string` type value. So to use `toFixed` it will have to be first converted to a `number` type.
2. `toFixed` pads the decimal side with additional zeroes if the number of digits in the number is less than the required number of digits. For example, 5.8 becomes 5.8000.

Thus with the current `truncateString` util function we are free from any type conversions and unnecessary zero padding of the decimal side.

Let me know what you think @dantaik ?